objects.go

```diff
@@ -300,6 +300,7 @@ type FixedAddress struct {
 	IPAddress   string `json:"ipv4addr,omitempty"`
 	Mac         string `json:"mac,omitempty"`
 	Name        string `json:"name,omitempty"`
+	MatchClient string `json:"match_client,omitempty"`
```

Comment:
Shouldn't there be a corresponding change in unit tests as a result of adding this field?

Reply:
just like name , even match client can be empty and not necessary to be passed.Hence did not add

Reply:
So, we should at least test the case when it IS passed in.

Reply:
okay, will add it